src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,58 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+	private static final int MIN_LOTTO_NUMBER = 1;
+	private static final int MAX_LOTTO_NUMBER = 45;
+
+	private final int value;
+
+	public LottoNumber(int value) {
+		validateLottoNumber(value);
+		this.value = value;
+	}
+
+	private void validateLottoNumber(int value) {
+		if (value < MIN_LOTTO_NUMBER || value > MAX_LOTTO_NUMBER) {
```

Comment:
사소하지만 아래 같이 조건식을 두면   `a < 변수 < b` 같은 구조처럼 보여서 조금 더 가독성이 좋아진다고 생각해요. 🙂
```suggestion
		if (MIN_LOTTO_NUMBER < value  || value > MAX_LOTTO_NUMBER) {
```

Reply:
위의 조건식은 `변수 < a(최소값)` or `b(최대값) < 변수`를 의미하는 것으로 유효한 범위를 벗어나는지 확인하는 조건식입니다.

이 경우 가독성을 고려하면 `변수 < a(최소값)` or `b(최대값) < 변수` 형태를 띄는 게 좋다고 생각해서 변경하였습니다 :)
https://github.com/next-step/java-lotto-kakao/pull/14/commits/42550360b101b825be29734457d36cfc6fbfa644

Reply:
에구... 죄송합니다.   이건 제가 로직을 잘못봐서 조금 엇나간 코멘트를 드렸네요. 🙏
혼동을 드려 죄송합니다.